types/options.go

```diff
@@ -179,6 +179,10 @@ func defaultStoreOptionsIsolated(rootless bool, rootlessUID int, storageConf str
 		storageOpts.RootlessStoragePath = storagePath
 	}
 
+	if storageOpts.ImageStore != "" && storageOpts.ImageStore == storageOpts.GraphRoot {
+		return storageOpts, fmt.Errorf("imagestore if set must be a different location than graphroot")
```

Comment:
It is set, o error should be.
return fmt..Errorf("imagestore %s must either be not set or be different then graphroot", storageOpts.ImageStore)

Reply:
updated.